Realm/RLMRealm.mm

```diff
@@ -294,6 +294,17 @@ + (NSString *)writeablePathForFile:(NSString*)fileName
     return [path stringByAppendingPathComponent:fileName];
 }
 
++ (NSString *)writeableTemporaryPathForFile:(NSString*)fileName
```

Comment:
`NSString*` -> `NSString *`.

`fileName` doesn't feel like the best name for the argument since it's not used as the file name. It's really an opaque identifier.